doc/nrf/app_dev/device_guides/nrf54l/nrf54l_signing_with_payload.rst

```diff
@@ -0,0 +1,238 @@
+.. _nRF54l_signing_app_with_flpr_payload:
+
+Signing applications with integrated FLPR payload
+#################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF54L SoCs include a FLPR CPU designed to handle fast communication with external devices independently, enhancing flexibility in protocol implementation.
+For details, see :ref:`vpr_flpr_nrf54l`.
+
+Users can dynamically load and modify the FLPR code during runtime.
+This guide describes how to build the application and FLPR cores, creating a single file, signed and managed by MCUboot, with FLPR integrated and executable from the application slot.
+
+Requirements and limitations
+****************************
+
+MCUboot requires the application and the FLPR application to be concatenated and signed together to treat them as a single image.
+To do this, you have to build the main application (CPUAPP application) and the FLPR application separately, manually merge them, and sign them for MCUboot.
+
+In addition to the requirements, there are also limitations caused by sysbuild.
+The FLPR application cannot be built as part of the main sysbuild process.
+Instead, the CPUAPP application can be built with MCUboot integration, while the FLPR must be built separately and then merged.
+For details, see :ref:`building_nrf54l`.
+Additionally, the DTS file for the CPUAPP application must allocate a specific reserved area for the FLPR binary.
```

Comment:
```suggestion
Note that the DTS file for the CPUAPP application must allocate a specific reserved area for the FLPR binary.
```

Reply:
Yeah, now I have noticed that I have written  "must specify specific specified area"